kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -263,11 +263,22 @@ def user_defined_tests(self):
         self.logger.info('Running user defined data validation tests '
                          f'{user_defined_test_dir} ...')
 
-        passed = pytest.main([user_defined_test_dir]) == 0
-
-        if passed:
-            self.logger.info(f'✅  User defined data validation tests passed!')
+        exit_code = pytest.main([user_defined_test_dir])
+
+        # Check if exit code is one of the codes that should result in pass
+        # Exit code 0:	All tests were collected and passed successfully
+        # Exit code 5:	No tests were collected
+        passed = exit_code in {0, 5}
+
+        if exit_code == 0:
+            self.logger.info(f'✅  User defined data validation tests passed')
+        elif exit_code == 5:
+            self.logger.warning(
+                f'⚠️  pytest did not collect any user defined tests, '
```

Comment:
Lol, its weird it doesn't properly render the unicode. Should be ⚠️

Reply:
??? 
![Screen Shot 2019-04-29 at 6 50 38 PM](https://user-images.githubusercontent.com/201996/56931911-b92bfe80-6aaf-11e9-9154-3dea2cdb6db4.png)

Reply:
whaaaat weird! 
![image](https://user-images.githubusercontent.com/10283712/56931977-f5f7f580-6aaf-11e9-8d2d-cddeaf92b49a.png)